Recipes/Recipes/Controllers/MainVC.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  MainViewController.swift
+//  Recipes
+//
+//  Created by Nick Nguyen on 2/10/20.
+//  Copyright © 2020 Lambda Inc. All rights reserved.
+//
+
+import UIKit
+
+class MainVC: UIViewController , UISearchBarDelegate
+{
```

Comment:
This curly brace needs to be at the end of line 11.
EX: 
`class MainVC: UIViewController, UISearchBarDelegate {`